wrench/reftests/filters/filter-grayscale.yaml

```diff
@@ -1,12 +1,19 @@
 ---
 root:
   items:
+    -
```

Comment:
It seems like this test doesn't actually have to change? If you are trying to test a different behavior, maybe it would be better to rely only on a new test?